MParT/Utilities/LinearAlgebra.h

```diff
@@ -270,6 +272,53 @@ class PartialPivLU
 
 };
 
+/** Mimics the interface of the Eigen::LLT class, but using Kokkos::Views and CUBLAS/CUSOLVER linear algebra.
+
+Note that the layout of the matrices used in this class is important.  Cublas expects column major (layout left).
+*/
+template<typename MemorySpace>
+class Cholesky
+{
+public:
+
+    Cholesky() {};
+    Cholesky(Kokkos::View<const double**,Kokkos::LayoutLeft,MemorySpace> A){ compute(A);};
+
+    /** Computes the Cholesky factorization of a matrix A */
+    void compute(Kokkos::View<const double**,Kokkos::LayoutLeft,MemorySpace> A);
+
+    /** Computes \f$A^{-1}x\f$ and stores the results in x.
+    */
+    void solveInPlace(Kokkos::View<double**,Kokkos::LayoutLeft,MemorySpace> x);
+
+    /** Returns a view containing \f$A^{-1}x\f$. */
+    Kokkos::View<double**,Kokkos::LayoutLeft,MemorySpace> solve(StridedMatrix<const double,MemorySpace> x);
+
+    /** Computes \f$L^{-1}B\f$ and stores the results in B.
+    */
+    void solveLInPlace(Kokkos::View<double**,Kokkos::LayoutLeft,MemorySpace> B);
```

Comment:
The `L` in the middle of the function name is not immediately obvious when I'm scanning the code.   What do you think about moving this to the end of the function name, so that we'd have `solveInPlaceL`?  Because `multiplyL`, has the `L` at the end, this seems more consistent and readable to me.